protocol-specs/generators/multisignature_registration/index.js

```diff
@@ -14,164 +14,206 @@
 
 'use strict';
 
-const {
-	hexToBuffer,
-	intToBuffer,
-	hash,
-	bufferToHex,
-	signData,
-} = require('@liskhq/lisk-cryptography');
+const { signData } = require('@liskhq/lisk-cryptography');
+const { Codec } = require('@liskhq/lisk-codec');
 const BaseGenerator = require('../base_generator');
 
-const networkIdentifier =
-	'e48feb88db5b5cf5ad71d93cdcd1d879b6d5ed187a36b0002cc34e0ef9883255';
+const codec = new Codec();
```

Comment:
We have a default instance in the package already, should be used.

Reply:
same as above